contrib/nydus-test/functional-test/test_nydus.py

```diff
@@ -564,7 +564,7 @@ def test_pseudo_fs(nydus_anchor, nydus_image: RafsImage, rafs_conf: RafsConf):
     nc.umount_rafs("/pseudo3")
 
 
-def test_shared_blobcache(nydus_anchor: NydusAnchor, nydus_image, rafs_conf: RafsConf):
+def test_shared_blobcache(nydus_anchor: NydusAnchor, nydus_image, rafs_conf: RafsConf, tmp_path):
```

Comment:
We can't add a parameter to the test function if it is not a fixture. Did you successfully run the test?

Reply:
Yes,I run these test functions successfully . But I'm not familiar with pytest especially the fixture, so I'm not sure.